opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/entrypoint/pis/Xs2aGetPaymentInfoEntrypoint.java

```diff
@@ -0,0 +1,98 @@
+package de.adorsys.opba.protocol.xs2a.entrypoint.pis;
+
+import de.adorsys.opba.protocol.api.common.ProtocolAction;
+import de.adorsys.opba.protocol.api.dto.context.ServiceContext;
+import de.adorsys.opba.protocol.api.dto.request.payments.PaymentInfoBody;
+import de.adorsys.opba.protocol.api.dto.request.payments.PaymentInfoRequest;
+import de.adorsys.opba.protocol.api.dto.result.fromprotocol.Result;
+import de.adorsys.opba.protocol.api.dto.result.fromprotocol.ok.SuccessResult;
+import de.adorsys.opba.protocol.api.pis.GetPaymentInfoState;
+import de.adorsys.opba.protocol.api.services.scoped.consent.ProtocolFacingConsent;
+import de.adorsys.opba.protocol.xs2a.context.pis.Xs2aPisContext;
+import de.adorsys.opba.protocol.xs2a.entrypoint.ExtendWithServiceContext;
+import de.adorsys.opba.protocol.xs2a.entrypoint.helpers.UuidMapper;
+import de.adorsys.opba.protocol.xs2a.service.dto.ValidatedQueryHeaders;
+import de.adorsys.opba.protocol.xs2a.service.mapper.QueryHeadersMapperTemplate;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.dto.DtoMapper;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.dto.payment.PaymentInfoHeaders;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.dto.payment.PaymentInfoParameters;
+import de.adorsys.xs2a.adapter.service.PaymentInitiationService;
+import de.adorsys.xs2a.adapter.service.Response;
+import de.adorsys.xs2a.adapter.service.model.SinglePaymentInitiationInformationWithStatusResponse;
+import lombok.RequiredArgsConstructor;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.springframework.stereotype.Service;
+
+import java.util.concurrent.CompletableFuture;
+
+import static de.adorsys.opba.protocol.xs2a.constant.GlobalConst.SPRING_KEYWORD;
+import static de.adorsys.opba.protocol.xs2a.constant.GlobalConst.XS2A_MAPPERS_PACKAGE;
+
+/**
+ * Entry point to get payment information. BPMN engine and process is not touched.
+ */
+@Service("xs2aGetPaymentInfoState")
+@RequiredArgsConstructor
+public class Xs2aGetPaymentInfoEntrypoint implements GetPaymentInfoState {
+    public static final String SEPA_CREDIT_TRANSFERS = "sepa-credit-transfers";
+    private final PaymentInitiationService pis;
+    private final PaymentInformationToBodyMapper mapper;
+    private final Xs2aGetPaymentInfoEntrypoint.Extractor extractor;
+    private final Xs2aGetPaymentInfoEntrypoint.FromRequest request2ContextMapper;
+    private final ExtendWithServiceContext extender;
+
+    @Override
+    public CompletableFuture<Result<PaymentInfoBody>> execute(ServiceContext<PaymentInfoRequest> context) {
+        ProtocolFacingConsent consent = context.getRequestScoped().consentAccess().findByCurrentServiceSession()
```

Comment:
```
.findByCurrentServiceSession()
                .orElseThrow(() -> new IllegalStateException("Context not found"));
```
can be replaced with 
ProtocolFacingConsent getByCurrentSession() {
.findByCurrentServiceSession()
                .orElseThrow(() -> new IllegalStateException("Context not found"));
}
inside ConsentAccess to avoid constant duplication